src/sub-blocks/Content/Content.tsx

```diff
@@ -43,23 +43,36 @@ function getButtonSize(size: ContentSize) {
     }
 }
 
-export type ContentProps = ContentBlockProps & ClassNameProps & QAProps;
+export type ContentProps = ContentBlockProps &
+    ClassNameProps &
+    QAProps & {
+        titleClassName?: string;
```

Comment:
for what purpose we need all of this customisations, mb it is easier to create own block, than redefine everyting
anyway in this case we should create only one object property for all of this classes and remove it from top level properties

Reply:
The idea was to reuse this component in custom block. And some of the inner components should be rearranged (paddings should be changed a little). Maybe for that case it's better to split the Content components into a smaller pieces that render text/links/buttons.